lib/ios-deploy.js

```diff
@@ -52,6 +52,10 @@ class IOSDeploy {
       return false;
     }
   }
+
```

Comment:
Yeah. The non-lazy thing to do would be to just update all the uses of the old one, which doesn't really need to exist. We control both sim and real device interfaces. ;)